projects/angular/program-website/src/app/routes/home/home.component.html

```diff
@@ -0,0 +1,155 @@
+          Lorem ipsum dolor sit amet
+        </a>
+      </div>
+      <div class="project-link">
+        <a href="https://www.hasadna.org.il/consecteturadipiscingelit/">
+          consectetur adipiscing elit
+        </a>
+      </div>
+      <div class="project-link">
+        <a href="https://www.hasadna.org.il/venenatisacondimentumvitae/">
+          Venenatis a condimentum vitae
+        </a>
+      </div>
+      <div class="project-link">
+        <a href="https://www.hasadna.org.il/pellentesquehabitantmorbitristique/">
+          pellentesque habitant morbi tristique
+        </a>
+      </div>
+      <div class="project-link">
+        <a href="https://www.hasadna.org.il/consecteturadipiscing/">
+          consectetur adipiscing
+        </a>
+      </div>
+      <div class="project-link">
+        <a href="https://www.hasadna.org.il/utlaboreetdoloremagna/">
+          ut labore et dolore magna
+        </a>
+      </div>
+    </div>
+    <div class="read-more-button">
+      <a class="read-more-link" href="https://www.hasadna.org.il/projects/">
+        READ MORE
+      </a>
+    </div>
+  </div>
+  <div class="final-section">
+    <div class="participants-container">
+      <p class="participants-title">
+        Participants
+      </p>
+      <p class="participants-text">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Venenatis a
+        condimentum vitae sapien pellentesque habitant morbi tristique
+        senectus.
+      </p>
+      <div class="read-more-button">
+        <a class="read-more-link" href="https://www.hasadna.org.il/projects/">
+          READ MORE
+        </a>
+      </div>
+    </div>
+    <div class="image-3-container">
+      <img src="../assets/images/image-3.svg" class="image-3">
+      <div class="contact-section">
+        <p class="contact-title">
+          Contact Us
+        </p>
+        <div class="contact-paragraphs">
+          <p class="contact-text">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor <a href="https://www.hasadna.org.il/contact/">HERE</a>.
+          </p>
+          <p class="contact-text">
+            Venenatis a condimentum vitae sapien pellentesque habitant
+            morbi tristique senectus <a href="https://www.hasadna.org.il/contact/">HERE</a>.
+          </p>
+        </div>
+        <div class="contact-information">
+          <p class="contact-information-text">
+            Venenatis a condimentum vitae at:
+          </p>
+          <div class="contact-email">
+            <a href="ouremail22@gmail.com">
+              ouremail22@gmail.com
+            </a>
+          </div>
+          <div class="contact-phone">
+            <p>
+              012-3456789
+            </p>
+          </div>
+        </div>
+      </div>
+      <img src="../assets/images/image-4.svg" class="image-4">
```

Comment:
Why do you use `../`?

Reply:
Is it possible to use `@/`?
I don't know how.

Reply:
Nevermind. Just had to remove `../` from the `src` string.